vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ImageService.java

```diff
@@ -247,6 +260,63 @@ public void addCategory(IVSImage image, ImageCategory category) {
     public void removeCategory(IVSImage image, ImageCategory category) {
         image.getCategories().remove(category);
         imageRepo.save((VSImage) image);
+    }   
+    
+    /**
+     * Method to store an image in the file location
```

Comment:
what file location? this is confusing. the method only gets an image and a filename, right? nothing about location?